### DISCUSSION TITLE
Finally got time to finish my tool for calculating expected wounds from an Infinity the Game Face to Face roll, thanks to icepool!

### DISCUSSION THREAD

**Discussion by alexrecarey** [HighDiceRoller/icepool#146] [Show and tell]
Hi @HighDiceRoller, just wanted to thank you for taking the time to help me grok how to use icepool. I just uploaded the[ first version of my app](https://infinitythecalculator.com), made for fun and for the Infinity community. Still have some work to do on accesibility, UX and other issues, but I'm super happy on how it works and it would have been totally impossible without you. Thanks again!

**Comment by HighDiceRoller** [comment]
Fantastic! I'm loving the skeuomorphism of the dice.
